src/ui/ui_apps/ui_apps_mixin.js

```diff
@@ -29,4 +29,18 @@ export function uiAppsMixin(kbnServer, server) {
   server.decorate('server', 'getAllUiApps', () => kbnServer.uiApps.slice(0));
   server.decorate('server', 'getUiAppById', id => appsById.get(id));
   server.decorate('server', 'getHiddenUiAppById', id => hiddenAppsById.get(id));
+
+  const injectedVarProviders = [];
```

Comment:
Yeah, definitely intended to support multiple providers for the same appId so that plugins can inject vars into other applications. Down the road I intend to add support for a `*` appId that will allow us to remove most if not all instances of `uiExports.injectDefaultVars` and `uiExports.replaceInjectedVars`, but didn't want to muddy up this pr with that.